src/soundfile.js

```diff
@@ -904,11 +904,12 @@ class SoundFile {
   }
 
   /**
-   * Returns the current pitch of a sound file in the MIDI note.
+   * Returns the current pitch of a sound file as a MIDI note.
    *
    * @method getPitch
    * @for p5.SoundFile
-   * @return {Number}  Current pitch of the SoundFile.
+   * @return {Number}  Current pitch of the SoundFile.The default note is assumed to
+   *                   be 60(middle C).
```

Comment:
The words here are great! I think we just need a space before "The" and a space after "60" and then we're good to go!